hack/local-up-cluster.sh

```diff
@@ -595,7 +595,7 @@ function start_kubelet {
       priv_arg="--allow-privileged "
     fi
 
-    mkdir -p /var/lib/kubelet
+    sudo mkdir -p /var/lib/kubelet
```

Comment:
@alindeman in line 247, the CERT_DIR is created using:
`mkdir -p "${CERT_DIR}" &>/dev/null || sudo mkdir -p "${CERT_DIR}"`

Do you want to use the same pattern here?
`mkdir -p /var/lib/kubelet &>/dev/null || sudo mkdir -p /var/lib/kubelet`